src/utils/getComments.jsx

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const instance = axios.create({
+	baseURL: "https://nc-news-vvzg.onrender.com/api",
+	timeout: 10000,
+});
+
+export default function getComments(article_id) {
+	const params = article_id;
+
+	return instance
+		.get("/articles/" + params + "/comments")
+		.then((response) => {
+			return response.data;
+		})
+		.catch((err) => {
```

Comment:
Instead of logging error here, I suggest to display a message in the page instead to let the user know that something went wrong.